### PR TITLE
Remove the Client

### PR DESCRIPTION
**Pull request by aaron-siegel**
Removes `pxt.Client` and replaces it with global methods in the `pxt` module. Refactors tests and docs to use the new pattern.

### CODE REVIEW

**Comment by aaron-siegel** [comment]
This change is [Reviewable](https://reviewable.io/reviews/pixeltable/pixeltable/158)

File: docs/api/client.md [hunk 1]
```diff
@@ -1,31 +1,29 @@
 # Client
```
Comment (mkornacker) [inline docs/api/client.md:1]: Doesn't this whole file need to go? Or at least get renamed?

File: pixeltable/globals.py [hunk 2]
```diff
@@ -0,0 +1,461 @@
...
+from pixeltable.exprs import Predicate
+from pixeltable.iterators import ComponentIterator
+from pixeltable.metadata import schema
+
+_logger = logging.getLogger('pixeltable')
+
+
+def init() -> None:
+    """Initializes the Pixeltable environment."""
+    _ = Catalog.get()
```
Comment (mkornacker) [inline pixeltable/globals.py:22]: What about Env? FunctionRegistry?

File: pixeltable/tests/conftest.py [hunk 3]
```diff
@@ -34,25 +34,21 @@ def init_env(tmp_path_factory) -> None:
     # ensure this home dir exits
     shared_home.mkdir(parents=True, exist_ok=True)
     # this also runs create_all()
-    Env.get().set_up(echo=True)
+    Env.get().configure_logging(level=logging.DEBUG, to_stdout=True)
```
Comment (mkornacker) [inline pixeltable/tests/conftest.py:37]: What calls set_up() now?

File: pixeltable/tests/test_client.py [hunk 4]
```diff
@@ -7,16 +7,13 @@
 class TestClient:
```
Comment (mkornacker) [inline pixeltable/tests/test_client.py]: Rename file?

**Review by aaron-siegel — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pixeltable/pixeltable/158)* status: 0 of 46 files reviewed, 4 unresolved discussions (waiting on @mkornacker)

___
*[`docs/api/client.md` line 1 at r2](https://reviewable.io/reviews/pixeltable/pixeltable/158#gh-1594366796:-NxOTEdK6xGOomGEU22g:blpsllm) ([raw file](https://github.com/pixeltable/pixeltable/blob/0f0e5c1c66452c4ec5086d404b9f1cd3ea22adfe/docs/api/client.md#L1)):*
<details><summary><i>Previously, mkornacker (Marcel Kornacker) wrote…</i></summary><blockquote>

Doesn't this whole file need to go? Or at least get renamed?
</blockquote></details>

Yeah, I'm going to make a pass through all the docs to clean them up as a follow-up to this PR.
___
*[`pixeltable/globals.py` line 22 at r2](https://reviewable.io/reviews/pixeltable/pixeltable/158#gh-1594369940:-NxOTIN800ylTrEGg3eE:bbmn0l6) ([raw file](https://github.com/pixeltable/pixeltable/blob/0f0e5c1c66452c4ec5086d404b9f1cd3ea22adfe/pixeltable/globals.py#L22)):*
<details><summary><i>Previously, mkornacker (Marcel Kornacker) wrote…</i></summary><blockquote>

What about Env? FunctionRegistry?
</blockquote></details>

Env gets initialized as a byproduct of calling `Catalog.get()`. Does anything special need to be done for `FunctionRegistry`?
___
*[`pixeltable/tests/conftest.py` line 37 at r2](https://reviewable.io/reviews/pixeltable/pixeltable/158#gh-1594373749:-NxOTPuABTT5xEzknVI5:b-yq37ij) ([raw file](https://github.com/pixeltable/pixeltable/blob/0f0e5c1c66452c4ec5086d404b9f1cd3ea22adfe/pixeltable/tests/conftest.py#L37)):*
<details><summary><i>Previously, mkornacker (Marcel Kornacker) wrote…</i></summary><blockquote>

What calls set_up() now?
</blockquote></details>

`Env.get()`, the first time it is called.
___
*[`pixeltable/tests/test_client.py` line 7 at r2](https://reviewable.io/reviews/pixeltable/pixeltable/158#gh-1594376387:-NxOTTOsCSQyyevQbSQU:b-y6tuap) ([raw file](https://github.com/pixeltable/pixeltable/blob/0f0e5c1c66452c4ec5086d404b9f1cd3ea22adfe/pixeltable/tests/test_client.py#L7)):*
<details><summary><i>Previously, mkornacker (Marcel Kornacker) wrote…</i></summary><blockquote>

Rename file?
</blockquote></details>

As a follow-up.

**Review by aaron-siegel — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pixeltable/pixeltable/158)* status: 0 of 46 files reviewed, 4 unresolved discussions (waiting on @mkornacker)

___
*[`pixeltable/tests/test_client.py` line 7 at r2](https://reviewable.io/reviews/pixeltable/pixeltable/158#gh-1594376387:-NxOWcde1g823xjfeU_n:b-ep6lok) ([raw file](https://github.com/pixeltable/pixeltable/blob/0f0e5c1c66452c4ec5086d404b9f1cd3ea22adfe/pixeltable/tests/test_client.py#L7)):*
<details><summary><i>Previously, aaron-siegel (Aaron Siegel) wrote…</i></summary><blockquote>

As a follow-up.
</blockquote></details>

Actually- I just took care of this one now. There were only 2 tests in it, which I moved out into `test_table.py` and `test_function.py`.

I do want to punt on refactoring the docs into a separate PR though :)

**Review by mkornacker — APPROVED** [review]

**Pull request merged**